api/types.go

```diff
@@ -24,14 +24,25 @@ type DocDBResponse struct {
 	// https://docs.aws.amazon.com/sdk-for-go/api/service/docdb/#DBCluster
 	Cluster *docdb.DBCluster
 	// https://docs.aws.amazon.com/sdk-for-go/api/service/docdb/#DBInstance
-	Instances []*docdb.DBInstance
+	Instances []*docdb.DBInstance `json:",omitempty"`
+	Tags      []*Tag              `json:",omitempty"`
 }
 
 type Tag struct {
 	Key   *string
 	Value *string
 }
 
+// tagsInOrg returns true if there is a spinup:org tag and it matches our org
+func tagsInOrg(org string, tags []*Tag) bool {
+	for _, t := range tags {
+		if aws.StringValue(t.Key) == "spinup:org" && aws.StringValue(t.Value) == org {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
This is nice - i wonder if we should have an `inOrg()` standard function that ships in the template...  if we did, I wonder if we should have it be a method.

```golang
type Tags []*Tag

func (t *Tags) inOrg(org string) bool {
	for _, t := range tags {
		if aws.StringValue(t.Key) == "spinup:org" && aws.StringValue(t.Value) == org {
			return true
		}
	}
	return false
}
```

or something like it...  thoughts?

Reply:
ooo, a method on the struct, I like that!